Source/MpSyncWorkers.cs

```diff
@@ -63,6 +70,43 @@ private static void SyncGameCondition(SyncWorker sync, ref GameCondition gameCon
             }
         }
 
+        private static void SyncDesignation(SyncWorker sync, ref Designation designation)
+        {
+            if (sync.isWriting)
+            {
+                sync.Write(designation?.designationManager);
+
+                if (designation != null)
+                {
+                    sync.Write(designation.target);
+                    sync.Write(designation.def);
+                }
+            }
+            else
+            {
+                var manager = sync.Read<DesignationManager>();
+
+                if (manager != null)
+                {
+                    var target = sync.Read<LocalTargetInfo>();
+                    var def = sync.Read<DesignationDef>();
+
+                    if (target.HasThing)
+                        manager.DesignationOn(target.Thing, def);
```

Comment:
Shouldn't this assign to designation?

Reply:
Good catch, should be fixed.

I've had to re-make this code as I seemed to have lost the original, so I must have messed up when doing this.